src/components/Card.js

```diff
@@ -70,15 +82,15 @@ export const CardList = (props) => {
                 Articles Per Page |
               </li>
               <li className="list-inline-item">
-                <a href="/"> 10</a>
+                <Link to="" value={10} onClick={setPageLimit} style={{ color: firstIndicator ? '#007fff' : '' }}>10</Link>
```

Comment:
The behavior of these links, when selected, are wrong. Please refer to the mockups.